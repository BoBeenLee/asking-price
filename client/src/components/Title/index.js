import React from 'react';
import styled from 'react-emotion';

const Root = styled('h1') `
    text-align: center;
`;


const Title = ({ children, ...rest }) => {
    return (
        <Root {...rest}>
            {children}
        </Root>
    );
};

export default Title;