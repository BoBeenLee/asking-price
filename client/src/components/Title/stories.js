import React from "react";
import { storiesOf } from '@storybook/react';
import { action } from '@storybook/addon-actions';
import Title from "./";

storiesOf("Title", module)
    .add("with Title", () => (
        <Title>Title</Title>
    ));