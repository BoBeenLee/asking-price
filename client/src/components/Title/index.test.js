import React from 'react';
import renderer from 'react-test-renderer';

import Title from './';

describe('title', () => {
    it('renders correctly', () => {
        const tree = renderer
            .create(<Title>title</Title>)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});