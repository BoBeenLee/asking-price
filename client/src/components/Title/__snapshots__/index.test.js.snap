// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`title renders correctly 1`] = `
<h1
  className="css-xi606m e1p76gxu0"
>
  title
</h1>
`;
